src/components/MyKiva/MyKivaStats.vue

```diff
@@ -104,7 +104,7 @@ onMounted(() => {
 	apollo.query({ query: lendingStatsQuery })
 		.then(result => {
 			livesTouched.value = result.data?.my?.lendingStats?.lentTo?.borrowers?.totalCount ?? 0;
-			totalAmountLent.value = +(result.data?.my?.userStats?.amount_of_loans ?? 0);
+			totalAmountLent.value = result.data?.my?.userStats?.amount_of_loans ?? 0;
```

Comment:
What was this causing before?

Reply:
Is returning a NAN for more than 3 digits b/c including a comma.
Formatting is covered in `StatItem` component, so we don't need it there.

Reply:
Ah, good to know, nice find.